### PR TITLE
Task/UIFunctionality/userAuth

### PR DESCRIPTION
**Pull request by Ogunjembola**
User Authentication-> UI

### CODE REVIEW

File: app/build.gradle
Comment (delaroy) [inline app/build.gradle:1]: revert to the Java SDK to 1.8

File: app/src/main/java/com/example/tapshopping/core/ShopApplication.kt
Comment (delaroy) [inline app/src/main/java/com/example/tapshopping/core/ShopApplication.kt:1]: revert back @HiltAndroidApp

File: app/src/main/java/com/example/tapshopping/core/di/NetworkModule.kt
Comment (delaroy) [inline app/src/main/java/com/example/tapshopping/core/di/NetworkModule.kt:1]: take off comment

File: app/src/main/java/com/example/tapshopping/data/service/NetworkService.kt
Comment (delaroy) [inline app/src/main/java/com/example/tapshopping/data/service/NetworkService.kt:1]: give the body class a descriptive name and do away withe the use of dot

File: app/src/main/java/com/example/tapshopping/domain/ShoppingRepository.kt
Comment (delaroy) [inline app/src/main/java/com/example/tapshopping/domain/ShoppingRepository.kt:1]: same here

File: app/src/main/java/com/example/tapshopping/ui/viewmodel/RegisterViewModel.kt
Comment (delaroy) [inline app/src/main/java/com/example/tapshopping/ui/viewmodel/RegisterViewModel.kt:1]: fuse the loading into the resource

File: app/src/main/res/layout/fragment_login.xml
Comment (delaroy) [inline app/src/main/res/layout/fragment_login.xml:1]: don't hardcode strings use reference

Comment (delaroy) [inline app/src/main/res/layout/fragment_login.xml:1]: don't use linear layout use constraint layout all through

**Review by delaroy — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/tapshopping/core/di/NetworkModule.kt
Comment (Ogunjembola) [reply app/src/main/java/com/example/tapshopping/core/di/NetworkModule.kt:1]: Fixed

File: app/src/main/java/com/example/tapshopping/core/ShopApplication.kt
Comment (Ogunjembola) [reply app/src/main/java/com/example/tapshopping/core/ShopApplication.kt:1]: Fixed

File: app/src/main/java/com/example/tapshopping/data/service/NetworkService.kt
Comment (Ogunjembola) [reply app/src/main/java/com/example/tapshopping/data/service/NetworkService.kt:1]: Fixed

File: app/src/main/java/com/example/tapshopping/ui/viewmodel/RegisterViewModel.kt
Comment (Ogunjembola) [reply app/src/main/java/com/example/tapshopping/ui/viewmodel/RegisterViewModel.kt:1]: Fixed

File: app/src/main/res/layout/fragment_login.xml
Comment (Ogunjembola) [reply app/src/main/res/layout/fragment_login.xml:1]: Fixed

File: app/build.gradle
Comment (Ogunjembola) [reply app/build.gradle:1]: fixed

File: app/src/main/java/com/example/tapshopping/domain/ShoppingRepository.kt
Comment (Ogunjembola) [reply app/src/main/java/com/example/tapshopping/domain/ShoppingRepository.kt:1]: fixed

File: app/src/main/res/layout/fragment_login.xml
Comment (Ogunjembola) [reply app/src/main/res/layout/fragment_login.xml:1]: fixed

**Pull request merged**